specification/communication/data-plane/readme.md

```diff
@@ -24,6 +24,22 @@ For other options on installation see [Installing AutoRest](https://aka.ms/autor
 
 These are the global settings for the communicationservices.
 
+```yaml
+openapi-type: data-plane
+tag: package-2021-03-07
+```
+
+### Tag: package-2021-03-07
+
+These settings apply only when `--tag=package-2021-03-07` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-03-07'
+input-file:
+  - Microsoft.CommunicationServicesAdministration/stable/2021-03-07/phonenumbers.json
```

Comment:
Let's move the files to data-plane/Microsoft.CommunicationServicesPhoneNumbers/stable/2021-03-07